src/common/eeprom.cpp

```diff
@@ -348,35 +345,38 @@ eeprom_init_status_t eeprom_init(void) {
     static eeprom_init_status_t status = EEPROM_INIT_Undefined;
     if (status != EEPROM_INIT_Undefined)
         return status; //already initialized
-    uint16_t version;
-    uint16_t features;
     status = EEPROM_INIT_Normal;
     osSemaphoreDef(eepromSema);
     eeprom_sema = osSemaphoreCreate(osSemaphore(eepromSema), 1);
     st25dv64k_init();
 
-    version = variant8_get_ui16(eeprom_get_var(EEVAR_VERSION));
-    features = (version >= 4) ? variant8_get_ui16(eeprom_get_var(EEVAR_FEATURES)) : 0;
-    if ((version >= EEPROM_FIRST_VERSION_CRC) && !eeprom_check_crc32())
+    //read entire eeprom at once
+    eeprom_vars_t &eevars = eeprom_startup_vars(); //initialize internal static variable, first call must be done now - to ensure interrupt is enabled
```

Comment:
Are those comments here necessary? Are they understandable?

I can imagine:
```
initialize internal static variables
read entire eeprom at once
```
would go to `eeprom_startup_vars()` documentation.

And I completely don't understand:
`first call must be done now - to ensure interrupt is enabled`
Can you make it more clear? Or delete it?